app/services/monthly_district_data_service.rb

```diff
@@ -23,9 +22,10 @@ def report
 
   private
 
-  def section_labels
+  def section_row
     [
-      Array.new(12, nil),
+      # These just add empty spacer columns
+      Array.new(13, nil),
```

Comment:
do these numbers just correspond to the number of empty columns? If so, would be worth extracting to a constant I think...`EMPTY_SECTION_ROWS` ?

Reply:
I cleaned this up with a different approach entirely